includes/class-wc-stripe-api.php

```
@@ -737,4 +752,172 @@ public static function get_masked_secret_key(): string {
+			return $session;
+		} catch ( \Stripe\Exception\ApiErrorException $e ) {
+			WC_Stripe_Logger::error(
+				'Stripe SDK error: create checkout session',
+				[
+					'stripe_api_key' => self::get_masked_secret_key(),
+					'error_message'  => $e->getMessage(),
+				]
+			);
+			throw new WC_Stripe_Exception( $e->getMessage(), $e->getMessage() );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method create_checkout_session is not named in camelCase. (undefined)

(CamelCaseMethodName)